tests/static_lib/tests/static_lib.rs

```diff
@@ -0,0 +1,29 @@
+use test_static_lib::Microsoft::Web::WebView2::Win32::CompareBrowserVersions;
+use windows::{initialize_sta, Result};
+
+#[test]
+fn test_less_than() -> Result<()> {
+    initialize_sta()?;
+    let mut result: i32 = 0;
+    unsafe { CompareBrowserVersions("86.0.622.0", "86.0.623.0", &mut result)? };
```

Comment:
A little surprised that the Windows crate didn't turn `result` into a return value. Is the last parameter marked `in` and `out`?

Reply:
I'm surprised too, but it's not annotated in the WebView2.h header:
```cpp
STDAPI CompareBrowserVersions(PCWSTR version1, PCWSTR version2, int* result);
```
That comes straight from the IDL as a `cpp_quote`:
```idl
/// This method is for anyone want to compare version correctly to determine
/// which version is newer, older or same.  Use it to determine whether
/// to use webview2 or certain feature based upon version.  Sets the value of
/// result to `-1`, `0` or `1` if `version1` is less than, equal or greater
/// than `version2` respectively.  Returns `E_INVALIDARG` if it fails to parse
/// any of the version strings or any input parameter is `null`.  Directly use
/// the `versionInfo` obtained from
/// `GetAvailableCoreWebView2BrowserVersionString` with input, channel
/// information is ignored.

cpp_quote("STDAPI CompareBrowserVersions(PCWSTR version1, PCWSTR version2, int* result);")
```

Reply:
And that's why - if you can convince the parameter in the winmd to be `out` then it will transform this into `-> Result<u23>`.

Reply:
Might need to go back to `win32metadata` and spin up another PR to allow specifying a `--remap` response file in the `EmmitWinmd` build task for the SDK. I think it should be possible to inject an `[Out]` attribute on the parameter that way.